### PR TITLE
Added support for MS SQL server driver for PHP 7.0

### PR DESCRIPTION
**Pull request by markhilton**
compiled extension for sqlsrv.so driver for the workspace container

### CODE REVIEW

File: workspace/Dockerfile-70 [hunk 1]
```diff
@@ -394,6 +394,85 @@ RUN if [ ${INSTALL_LINUXBREW} = true ]; then \
 ;fi
 
 #####################################
+# SQL SERVER:
+#####################################
+ARG INSTALL_MSSQL=true
```
Comment (winfried-van-loon) [inline workspace/Dockerfile-70]: The `INSTALL_MSSQL` looks great here. It should be added on the following places to integrate it into Laradock:

- `docker-compose.yml` in the `workspace` > `build` > `args` section.
- `env-sample` in the `Containers Customization` > `WORKSPACE` section.

**Comment by winfried-van-loon** [comment]
Also, before merging, we may want to upgrade the sqlsrv version as mentioned in #843 to avoid that error recurring. This is related to #882 and #883.

**Comment by markhilton** [comment]
If I remember well I added:

cd / && \

because originally I added MS SQL support build after WORKDIR inside Dockerfile, therefore wget was downloading unixodbc libs into my application code base, as workdir is a volume mounted to local filesystem. It can be removed. I will clean this up and re-submit.

**Comment by markhilton** [comment]
I submitted updates for this pull requests but seems like build on Travis fails due to unrelated issue with missing XML for INSTALL_V8JS_EXTENSION

**Comment by winfried-van-loon** [comment]
Yep, that's open issue #792, not quite sure what to do about that, to be honest. Thanks for the updates to the pull request, will look at your changes very soon.

File: php-fpm/Dockerfile-70 [hunk 2]
```diff
@@ -312,6 +312,81 @@ RUN if [ ${INSTALL_GHOSTSCRIPT} = true ]; then \
...
+        && odbcinst -q -d -n "ODBC Driver 13 for SQL Server" && \
+
+
+    #####################################
+    # Install sqlsrv y pdo_sqlsrv
+    # extensions:
+    #####################################
+
+    pecl install sqlsrv-4.1.7 && \
+    pecl install pdo_sqlsrv-4.1.7 && \
```
Comment (winfried-van-loon) [inline php-fpm/Dockerfile-70]: Version of these should be increased

File: workspace/Dockerfile-70 [hunk 3]
```diff
@@ -508,6 +508,81 @@ RUN if [ ${INSTALL_SYMFONY} = true ]; then \
...
+        && odbcinst -q -d -n "ODBC Driver 13 for SQL Server" && \
+
+
+    #####################################
+    # Install sqlsrv y pdo_sqlsrv
+    # extensions:
+    #####################################
+
+    pecl install sqlsrv-4.1.7 && \
+    pecl install pdo_sqlsrv-4.1.7 && \
```
Comment (winfried-van-loon) [inline workspace/Dockerfile-70]: Version of these should be increased

File: php-fpm/Dockerfile-70 [hunk 4]
```diff
@@ -273,8 +273,8 @@ RUN if [ ${MSSQL} = true ]; then \
     # extensions:
     #####################################
 
-    pecl install sqlsrv-4.0.6 && \
-    pecl install pdo_sqlsrv-4.0.6 && \
+    pecl install sqlsrv-4.1.7preview && \
+    pecl install pdo_sqlsrv-4.1.7preview && \
```
Comment (winfried-van-loon) [inline php-fpm/Dockerfile-70]: Version number change reference

Comment (winfried-van-loon) [inline php-fpm/Dockerfile-70]: Or we should lower this to stable, now that I'm looking at it.

**Pull request merged**